examples/rapids/mlflow_project/src/rf_test/train.py

```diff
@@ -94,8 +94,13 @@ def train(params, fpath, hyperopt=False):
     fn = partial(train, fpath=args.fpath, hyperopt=True)
     experid = 0
 
+    artifact_path = "Airline-Demo"
+    artifact_uri = None
+
+    mlflow.set_tracking_uri(uri='sqlite:////tmp/mlflow-db.sqlite')
     with mlflow.start_run():
-        mlflow.set_tag("mlflow.runName", "RAPIDS-Hyperopt")
+        mlflow.set_tag("mlflow.runName", "(CLI) RAPIDS-Hyperopt")
```

Comment:
To set the run name, you can actually pass the `run_name` argument to `mlflow.start_run` ([see docs](https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.start_run))

Reply:
Maybe the name should just be `RAPIDS-Hyperopt`? Since the script isn't necessarily run via the CLI

Reply:
It just indicates which example the result came from, so people exploring can easily identify whether the run was logged from the CLI example or the Notebook.

Reply:
Ah got it, thanks for clarifying! Would it suffice to look at the source column in the UI for that? Ideally, the "source" would be what users look at to identify the source of the run (i.e. whether it's the CLI example or notebook)